src/util/log.h

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file
+ * @brief
+ *
+ * @date 18.06.15
+ * @author Vita Loginova
+ */
+
+#ifndef UTIL_LOG_H_
+#define UTIL_LOG_H_
+
+#include <framework/mod/self.h>
+#include <util/logging.h>
+
+struct logger {
+	const struct mod *mod;
+	struct logging logging;
+};
+
+#define LOG_DECLARE_LOGGER() \
+	extern struct logger mod_logger __attribute__ ((weak)); \
+
+#define log_logp(level, fmt, ...) \
+	logging_raw(&mod_logger.logging, level, "%s: %s: " fmt "\n", \
```

Comment:
in case if mod_logger is not defined (weak ref turns it to NULL), seems like its NULL reference